src/main/resources/worker.sh

```diff
@@ -62,6 +66,17 @@ cat /etc/aws
 echo AWS Log agent logs:
 cat /var/log/awslogs.log
 
+# Set up aws cli
```

Comment:
This comment made me expect at first to see it installing the CLI software (though that's already available on Amazon Linux machines). It might be better to say something like "Create a config file to tell the AWS CLI which region to operate in."

Reply:
👍